playgrounds/app/src/components/Editor.tsx

```diff
@@ -776,28 +872,52 @@ async function createAnimationFrame(
 ) {
   const { yPadding, xPadding } = config.layout
   const { shadowEnabled, shadowOffsetY, shadowBlur, shadowColor, shadowOpacity } = config.shadow
-  const { fontSize, fontFamily, backgroundColor, snippetBackgroundColor } = config.styling
+  const { fontSize, fontFamily, backgroundColor, snippetBackgroundColor, backgroundType, backgroundGradientColorStart, backgroundGradientColorEnd, backgroundGradientDirection } = config.styling
 
+  console.log({ backgroundColor, backgroundType, backgroundGradientColorStart, backgroundGradientColorEnd, backgroundGradientDirection })
   const canvas = document.createElement('canvas')
-  const ctx = canvas.getContext('2d', { alpha: false })
+  const ctx = canvas.getContext('2d', { alpha: false })!
```

Comment:
I think we should leave this as is and use ctx!.

We can make a whole new separate PR later to refactor the ctx!. But by doing it in this PR, the diff is too noisy and shows changes that aren't relevant to the goal of this PR.